src/Microsoft.DotNet.Wpf/src/PresentationFramework/Microsoft/Win32/CommonItemDialog.cs

```diff
@@ -0,0 +1,693 @@
+                // Avoid returning a null string - return String.Empty instead.
+                return _initialDirectory.Value == null ? String.Empty : _initialDirectory.Value;
+            }
+            set
+            {
+
+                _initialDirectory.Value = value;
+            }
+        }
+
+        /// <summary>
+        ///  Restores the current directory to its original value if the user
+        ///  changed the directory while searching for files.
+        ///
+        ///  This property is only valid for SaveFileDialog;  it has no effect
+        ///  when set on an OpenFileDialog.
+        /// </summary>
+        public bool RestoreDirectory
+        {
+            get
+            {
+                return GetOption(FOS.NOCHANGEDIR);
+            }
+            set
+            {
+
+                SetOption(FOS.NOCHANGEDIR, value);
+            }
+        }
+
+        /// <summary>
+        ///       Gets or sets a string shown in the title bar of the file dialog.
+        ///       If this property is null, a localized default from the operating
+        ///       system itself will be used (typically something like "Save As" or "Open")
+        /// </summary>
+        public string Title
+        {
+            get
+            {
+                // Avoid returning a null string - return String.Empty instead.
+                return _title.Value == null ? String.Empty : _title.Value;
+            }
+            set
+            {
+
+                _title.Value = value;
+            }
+        }
+
+        //   If false, the file dialog boxes will allow invalid characters in the returned file name. 
+        //   We are actually responsible for dealing with this flag - it determines whether all of the
+        //   processing in ProcessFileNames (which includes things such as the AddExtension feature)
+        //   occurs.
+        /// <summary>
+        ///  Gets or sets a value indicating whether to check for situations that would prevent
+        ///  an application from opening the selected file, such as sharing violations or access denied errors.
+        /// </summary>
+        public bool ValidateNames
+        {
+            get
+            {
+                return !GetOption(FOS.NOVALIDATE);
+            }
+            set
+            {
+
+                SetOption(FOS.NOVALIDATE, !value);
+            }
+        }
+
+        public IList<FileDialogCustomPlace> CustomPlaces { get; set; }
+
+        #endregion Public Properties
+
+        //---------------------------------------------------
+        //
+        // Protected Methods
+        //
+        //---------------------------------------------------
+        #region Protected Methods
+
+        /// <summary>
+        /// Handles the IFileDialogEvents.OnFileOk callback.
+        /// </summary>
+        protected virtual void OnItemOk(CancelEventArgs e) { }
+
+        //  Because this class, FileDialog, is the parent class for both OpenFileDialog
```

Comment:
Comment needs to be updated here.

Reply:
Fixed in https://github.com/dotnet/wpf/pull/7916/commits/543c8098a03629f522d8d86a9666b673a0e0a045